Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:bookworm
+
+RUN apt-get update && apt install -y zip wget curl golang build-essential \
+    pkg-config libaio-dev libblkid-dev libkeyutils-dev \
+    liblz4-dev libsodium-dev liburcu-dev libzstd-dev \
+    uuid-dev zlib1g-dev valgrind libudev-dev udev git \
+    python3 python3-docutils libclang-dev debhelper dh-python 
+
+# install rust
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+ENV SOURCE_DATE_EPOCH 1600000000
+
+# build bcachefs-tools
+RUN mkdir /src /bins && wget https://github.com/koverstreet/bcachefs-tools/archive/refs/heads/master.zip && unzip -d /src /master.zip && mv /src/bcachefs-tools-master/* /src
```

Comment:
Can you change this to use a tagged version instead of the master branch? With e2fsprogs, we also track a released version.

Reply:
Done